register.html

```diff
@@ -35,28 +35,30 @@
                                 alt="lazy sloth logo">
                         </div>
 
-                        <form class="d-flex flex-column gap-2">
-                            <div>
+                        <form class="d-flex flex-column gap-2" id="form">
 
+                            <div>
                                 <div class="form-floating mb-3">
-                                    <input type="email" class="form-control" id="userEmail" aria-describedby="emailHelp"
-                                        placeholder="Email" required>
+                                    <input name="useremail" type="email" class="form-control" id="userEmail"
+                                        placeholder="Email" required
+                                        pattern="[\w\.%+-]+@(stud\.){0,1}noroff\.no">
                                     <label for="userEmail">Email</label>
-                                    <div class="errorMessage"></div>
+                                    <p id="regInfoText" class="text-black opacity-50 subText ms-2">Only noroff.no emails are allowed to register</p>
```

Comment:
```suggestion
```
It's best to let `title=""` handle the task of showing the user a message instead of custom JS. Form handling is a standard and best accessibility is achieved when it is preferred over other options.